manifests/params.pp

```diff
@@ -22,7 +22,10 @@
       $virtinst_package = 'virtinst'
       $radvd_package = 'radvd'
       $sysconfig = false
-      $deb_default = {}
+      $deb_default = $::service_provider ? {
+        'systemd' => { 'libvirtd_opts' => '' },  # no '-d', it confuses systemd
```

Comment:
Interesting this could change with the Puppet distribution. I'm running the latest puppet 4.6.0. I did not run with Puppet 3. Do you have this issue with Puppet 3? According to the debian provider it uses a init based system (which is super weird Puppet 3 report that on Ubuntu 16). I'll try to install a new machine with Ubuntu 16 + Puppet 3 and dig into it.